README.md

```diff
@@ -167,6 +167,50 @@ you modify the JVM's [networkaddress.cache.ttl
 property](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/net/doc-files/net-properties.html)
 to `60` seconds.
 
+### How to use undocumented parameters and properties
+
+stripe-java is a typed library and it supports all public properties or parameters.
+
+Stripe sometimes has beta which introduces new properties or parameters that are not immediately public. The library does not support these properties or parameters until they are public but there is still an approach that allows you to use them.
```

Comment:
We do have beta packages now. @pakrym-stripe can tell more.

Reply:
Yeah I think we should defer that mention to later, it's quite new and will only support a subset of betas, but curious how you plan to document this!